### ISSUE TITLE
Get current config

### ISSUE THREAD

**Issue by iBicha** [google-ar/arcore-android-sdk#505]
Is it possible to get the current configuration running in a given session?

**Comment by inio** [comment]
Yep!  Use [`Session.getConfig`](https://developers.google.com/ar/reference/java/com/google/ar/core/Session.html#getConfig\(\)) and [`ArSession_getConfig`](https://developers.google.com/ar/reference/c/group/session#group__session_1ga6e6d889878d87ed82b853b5cdfccab2a)
